### PR TITLE
fix(RangeInput): allow precision of 0

### PR DESCRIPTION
**Pull request by tkrugg**
**Summary**

Prior to this fix it was not possible to use precision 0

```html
<ais-range-input attribute="price" precision="0"></ais-range-input>
```

And so it was not possible to produce steps of size `1`.

Closes https://github.com/algolia/angular-instantsearch/issues/374

### CODE REVIEW

**Comment by Haroenv** [comment]
why 0 and not 1?

**Comment by francoischalifour** [comment]
Definition of `precision`:

> The number of digits after the decimal point to use.

[See documentation →](https://www.algolia.com/doc/api-reference/widgets/range-slider/js/#widget-param-precision)

**Comment by tkrugg** [comment]
we should maybe make default precision 0 instead of 2 to match the default value we have in vanilla ?

**Comment by francoischalifour** [comment]
Indeed. In `connectRange`, the [default value is 2](https://github.com/algolia/instantsearch.js/blob/56ad416f0d05aa59721c05c9c8c0731b36bffb4a/src/connectors/range/connectRange.js#L52), but in the `rangeInput` widget, the [default value is 0](https://github.com/algolia/instantsearch.js/blob/develop/src/widgets/range-input/range-input.js).

**Comment by samouss** [comment]
@francoischalifour @tkrugg Yes it's something that we should have done inside the V3 of InstantSearch but we forget. Never-mind. The default value should `0` indeed.

File: src/range-input/range-input.ts [hunk 1]
```diff
@@ -77,7 +77,10 @@ export class NgAisRangeInput extends BaseWidget {
   public maxInputValue?: number | string = '';
 
   get step() {
-    const precision = parseNumberInput(this.precision) || 2;
+    const precision =
+      typeof this.precision !== 'undefined'
+        ? parseNumberInput(this.precision)
+        : 2;
```
Comment (samouss) [inline src/range-input/range-input.ts]: We can use the call property to define the default value no? Or is it provided as string? 

Anyway the default value should be `0`.

**Pull request merged**